classes/popups/class-settings.php

```diff
@@ -0,0 +1,72 @@
+<?php
+/**
+ * Settings popup.
+ *
+ * @package Progress_Planner
+ */
+
+namespace Progress_Planner\Popups;
+
+use Progress_Planner\Activities\Content_Helpers;
+
+/**
+ * Settings popup.
+ */
+final class Settings extends Popup {
+
+	/**
+	 * The popover ID.
+	 *
+	 * @var string
+	 */
+	protected $id = 'settings';
+
+	/**
+	 * Render the triggering button.
+	 *
+	 * @return void
+	 */
+	public function render_button() {
+		?>
+		<!-- The triggering button. -->
+		<button class="prpl-info-icon" popovertarget="prpl-popover-<?php echo \esc_attr( $this->id ); ?>">
+			<span class="dashicons dashicons-admin-generic"></span>
+			<span class="screen-reader-text"><?php \esc_html_e( 'Settings', 'progress-planner' ); ?>
+		</button>
+		<?php
+	}
+
+	/**
+	 * Render the widget content.
+	 *
+	 * @return void
+	 */
+	protected function the_content() {
+		$saved_settings = Content_Helpers::get_post_types_names();
+		$post_types     = \array_filter( \get_post_types( [ 'public' => true ] ), 'is_post_type_viewable' );
+		unset( $post_types['attachment'] );
+		unset( $post_types['elementor_library'] ); // Elementor templates are not a post type we want to track.
+		?>
+
+		<h2><?php \esc_html_e( 'Settings', 'progress-planner' ); ?></h2>
+
+		<form id="prpl-settings-form">
+			<h3><?php \esc_html_e( 'Post Types', 'progress-planner' ); ?></h3>
+			<p><?php \esc_html_e( 'Select the post types you want to include in activity scores. This setting will affect which post-type activities get tracked.', 'progress-planner' ); ?></p>
+			<?php foreach ( $post_types as $post_type ) : ?>
+				<label>
+					<input
+						type="checkbox"
+						name="prpl-settings-post-types-exclude[]"
```

Comment:
Ok I'm going to be annoying, we should change this to include to prevent future confusion :)